roles/gateway_update_otk_hostnames/README.md

```diff
@@ -0,0 +1,52 @@
+Update Hostnames used by OTK
+============================
+
+This role updates the hostnames used by OTK using the values configured under this role's `vars/main.yml`:
+- Gateway cluster.hostname cluster property
+- OTK's JDBC connection URLs
+- Callback URLs
+
+Requirements
+------------
+
+Before running this role:
+- OTK should already be migrated to the destination database, which is the internal MySQL database running on a Gateway.
+- Ensure the Ansible controller can modify the OTK database
+  - Note: otk variables correspond to the value of the otkdb_name, otkdb_user and otkdb_userpwd variables, respectively (see Group Variables section)
+  - e.g. CREATE USER <otk user>@'localhost' IDENTIFIED BY '<otk user pass>';
+         GRANT SELECT, INSERT, DELETE, UPDATE ON <otk db name>.* TO <otk user>@'localhost';
+
+After running this role:
+- **Clean up grants after this role is executed.**
+- Follow the steps for [Set a Default SSL or CA Private Key](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/security-configuration-in-policy-manager/tasks-menu-security-options/manage-private-keys/set-a-default-ssl-or-ca-private-key.html) for the updated hostname.
```

Comment:
Not clear about what this step is trying to say.  How about:

To update the hostname of a certificate in the Certificate manager, please follow (link).